axolotl-web/src/components/Message.vue

```diff
@@ -1,96 +1,71 @@
 <template>
-  <div
-    :key="message.ID"
-    :class="{
-      'col-12': true,
-      'message-container': true,
-      outgoing: message.Outgoing,
-      sent: message.IsSent && message.Outgoing,
-      read: message.IsRead && message.Outgoing,
-      delivered: message.Receipt && message.Outgoing,
-      incoming: !message.Outgoing,
-      status:
-        (message.Flags > 0 &&
-          message.Flags !== 11 &&
-          message.Flags !== 13 &&
-          message.Flags !== 14) ||
-        message.StatusMessage ||
-        (message.Attachment &&
-          message.Attachment.includes('null') &&
-          message.Message === ''),
-      hidden: message.Flags === 18,
-      error: message.SentAt === 0 || message.SendingError,
-      'group-message':
-        isGroup && (message.Flags === 0 || message.Flags === 12 || message.Flags === 13),
-    }"
-  >
-    <div
-      v-if="
-        !message.Outgoing &&
-          isGroup &&
-          (message.Flags === 0 || message.Flags === 12 || message.Flags === 13)
-      "
-      class="avatar"
-    >
+  <div :key="message.ID" :class="{
```

Comment:
`message.ID` vs `chat.id` can we make this consistent?

Reply:
The message has no field ID anymore, this is still a clean up in progress